### ISSUE TITLE
Gaussian noise was zeroed in b

### ISSUE THREAD

**Issue by Kirija** [XPIR-team/XPIR#37]
The code in NFLLWE::recomputeNoiseAmplifiers() was messed up resulting in no noise at all in the b part of ciphers.

it did : 
```
tmpz1=0
tmpz2=0
foreach modulus
   tmpz2=2**Abits
   tmpz2=modulus
   Abitmod = tmpz1 % tmpz2
endforeach
```
I guess the point was to do Abitmod = 2**Abits % modulus and that "tmpz2=2**Abits" was meant to be "tmpz1=2**Abits"

PR coming with this correction.

**Comment by carlosaguilarmelchor** [comment]
Hey @Kirija do you think you can do that PR ?

**Comment by carlosaguilarmelchor** [comment]
Closed by PR#38
